scripts/create-aks-load-balancer-publicip-prefix.sh

```diff
@@ -9,4 +9,5 @@ CLUSTER_NAME=$2
 #az network public-ip delete --name ${CLUSTER_NAME}-pip --resource-group ${RESOURCE_GROUP_NAME}
 
 #creating it ahead of aks arm template as arm template doesn't validate very well.
-az network public-ip prefix create --length 28 --name ${CLUSTER_NAME}-pip --resource-group ${RESOURCE_GROUP_NAME}
+az upgrade --yes
```

Comment:
bit risky =/, can we upgrade to a specific version or install a specific version instead?

Reply:
Was going to be temporary, 2.21.0 only came out yesterday.

Will check again end of week, hopefully self-hosted agents be updated shortly to 2.21.0
https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/docs-ref-conceptual/release-notes-azure-cli.md#network

Reply:
can you add a comment to the script and set a reminder too?